Gemfile

```diff
@@ -8,6 +8,8 @@ end
 gem 'rails', '~> 5.1'
 
 gem 'autoprefixer-rails'
+gem 'axlsx', '2.1.0.pre'
```

Comment:
Can we relax the version requirement a bit, e.g. `~> 2.1.0`?

Also I always try to add a comment when using a specific version to explain why (e.g. with a link to a relevant bug report or something)